### DISCUSSION TITLE
Flusso operazioni ANSC

### DISCUSSION THREAD

**Discussion by Sem98** [italia/ansc#700] [Supporto servizi cooperativi] [answered]
### Ambiente

pre-produzione

### Servizio

R010 - Servizi cooperativi anteprima

### idOperazione

assente

### Provenienza segnalazione

Software House

### Comune di riferimento e operatore

Arezzo

### Descrivi il problema

Buonasera volevamo chiedere se il flusso delle operazioni da eseguire su ANSC fosse corretto. Noi eseguimo in sequenza R009 (con eventuali R001 riguardanti l'invio degli allegati)-R010-R006-R007. 
Volevamo chiedere se per il caso della firma cartacea si debba scaricare l'anteprima mediante R010 prima di inviarla firmata tramite R006 poichè il file generato dalla R010 contiene la scritta anteprima e l'assenza di campi utilizzabili per la firma dei dichiaranti.

**Comment by maxill1** [comment, marked as answer]
Il flusso descritto è corretto. 

Per quanto riguarda l'anteprima da firmare con il servizio R010, valorizzando `"presaVisione": true` il servizio restituirà il documento senza il watermark con i campi utilizzabili per la firma.

Per ulteriori dettagli si può fare riferimento alle specifiche del servizio R010:
* https://github.com/italia/ansc/blob/main/docs/openapi/R010_anteprima.yaml
* https://github.com/italia/ansc/blob/main/docs/Caratteristiche_servizi/R010_anteprima.pdf

Saluti

Assistenza ANSC

P.S. Trattandosi di un quesito converto la issue in discussion.

**Comment by Sem98** [comment]
Abbiamo verificato che cambiando il valore di presavisione i watermark sono scomparsi, grazie per l'aiuto.
